pwndbg/commands/got.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import pwndbg.chain
+import pwndbg.commands
+import pwndbg.enhance
+import pwndbg.file
+import pwndbg.which
+import pwndbg.wrappers
+
+from pwndbg.color import green
+from pwndbg.color import light_yellow
+
+
+@pwndbg.commands.Command
+@pwndbg.commands.OnlyWhenRunning
+@pwndbg.commands.OnlyWithFile
+def got():
+    '''
+    Show the state of the Global Offset Table
+    '''
+
+    local_path = pwndbg.file.get_file(pwndbg.proc.exe)
+    cs_out = pwndbg.wrappers.checksec("--file", local_path)
+
+    file_out = pwndbg.wrappers.file(local_path)
+    if "statically" in file_out:
+        return "Binary is statically linked."
+
+    readelf_out = pwndbg.wrappers.readelf("-r", local_path)
+
+    jmpslots = '\n'.join(filter(lambda l: _extract_Jumps(l),
+                         readelf_out.splitlines()))
+
+    if not len(jmpslots):
+        return "NO JUMP_SLOT entries available in the GOT"
+
+    if "PIE enabled" in cs_out:
+        bin_text_base = pwndbg.memory.page_align(pwndbg.elf.entry())
+
+    relro_status = "No RELRO"
+    if "Full RELRO" in cs_out:
+        relro_status = "Full RELRO"
+    elif "Partial RELRO" in cs_out:
+        relro_status = "Partial RELRO"
+
+    print("\nGOT protection: %s | GOT functions: %d\n " % (green(relro_status), len(jmpslots.splitlines())))
+
+    if pwndbg.arch.ptrsize == 4:
```

Comment:
It seems for me that both branches differs only in `line.split()` and maybe PIE (if x86-32 doesn't do anything with PIE).

Maybe:
```python
for line in jmpslots.splitlines():
    address, info, rtype, value, name = line.split()[:5]  # x86-64 has two columns redundant for us 
```